### ISSUE TITLE
Too many "read from buffer" messages running launchIOSDevice

### ISSUE THREAD

**Issue by odbuser2** [javafxports/javafxmobile-plugin#46]
org.javafxports:jfxmobile-plugin:2.0.18
OSX
Gradle 4.6
Oracle JDK 9.0.4

There are a lot of output lines when running launchIOSDevice:
```
read from buffer: 450
read from buffer: 1140
read from buffer: 450
```
Can these be combined or put behind a switch?

**Comment by johanvos** [comment]
should be fixed with 2.0.19

**Comment by odbuser2** [comment]
Confirmed.  Closing.
